sqlstore/src/main/resources/db/migration/V0028__model_refactor.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE SampleAdditionalInfo ADD COLUMN groupId int(10);
+ALTER TABLE SampleAdditionalInfo ADD COLUMN groupDescription varchar(255);
+
+UPDATE SampleAdditionalInfo SET groupId = (SELECT groupId FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId), groupDescription = (SELECT groupDescription FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId);
```

Comment:
Yes, they will be removed in GLT-746 where Analyte will be split into Stock and Aliquot.